README.adoc

```diff
@@ -75,7 +79,7 @@ This config is compatible with {uri-lazyvim}[LazyVim].
     "L3MON4D3/LuaSnip",
   },
   build = function()
-    require('pkl-neovim.internal').init()
+    require('pkl-neovim').init()
```

Comment:
In the interests of making this copy-pasteable, worth including the start_command setup below? This worked for me:

```lua
    config = function()
      require("luasnip.loaders.from_snipmate").lazy_load()
      vim.g.pkl_neovim = {
        start_command = { "pkl-lsp" },
      }
    end,
```

Reply:
yup, good idea!